.github/workflows/CI.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
```

Comment:
Ah, I added this before I knew how to update the Dockerfile. If the tests in the Dockerfile are actually executed on every push, we might not need this.